gulp/tasks/styles.js

```diff
@@ -43,13 +43,13 @@ function stylesModern() {
 }
 
 /**
- * Process legacy CSS for IE9 only.
+ * Process legacy CSS for IE9 and below.
  * @returns {PassThrough} A source stream.
  */
-function stylesIE9() {
+function stylesIE() {
```

Comment:
Should we name this `stylesLegacyIE` so it's clear it doesn't do anything to IE 10 or 11?

Reply:
Technically 10/11 are legacy too 😁

Reply:
Well shoot...